### ISSUE TITLE
Branch: Phase

### ISSUE THREAD

**Issue by annabelleee** [archesproject/ARM_Working_Group#27]
Steps to completion (updated):
- [ ] Metadata
- [ ] Diagram (using ARM WG notation)
- [ ] Sample Data
- [ ] Sample Concepts
- [ ] JSON of Arches branch based on diagram in ARM WG github repo
- [ ] Vote in comments to officially approve branch

**Comment by azaroth42** [comment]
Can we do this?

<img width="595" alt="state_phase" src="https://user-images.githubusercontent.com/871868/44883587-7984ec80-ac6c-11e8-8e9d-92ba73b62759.png">

Phase would be S16 State. The starting event is from O14. There isn't an end-of-phase relationship so we create the parallel la:terminated.  Per discussion, we have the entity that it is a state of, the relationship and related entity that define the state (if possible), and the regular fields of label, type, name, identifier, etc etc.

**Comment by azaroth42** [comment]
Per slack ... @Habennin says:
> As to State, you’ll see that in the latest edition of CRMsci, it has been partially removed. The idea of phase and social fact or something are good candidates for replacement however.  

So will replace S16 and O14 with linked.art terms.

**Comment by Habennin** [comment]
So I wrote something up here to discuss amongst ourselves what is the right solution

https://docs.google.com/document/d/1dLFbqChcnXlMbP7h7C9KDDX3leAmVNCGMFgT54XnRgo/edit?usp=sharing

It was complicated so I put in a google doc.

About E4, I am not sure. What it gives us is the ability to talk about place. But when we are talking about the states is what we really want to do not just to express the limited duration of it 'being the case' that such and such an attribute holds of such and such a thing?

**Comment by annabelleee** [comment]
Rob's document: https://docs.google.com/document/d/1OfQfHf34wEq8RaC-vTcKlmQpADoodq_GcCRF5wGitxo/edit?ts=5b8eb4ab

**Comment by azaroth42** [comment]
Discussion on ARM call 2018-09-13 - Stick with E2 not E4, as the spatial extent of a Phase is semantically strange in many situations. Concern about using Phase for the relationship between two Temporal Entities (it is meaningless to say that World War Two consisted of Battle of Normandy between 1980 and 1985) however there are relationships between non Persistent Items that do make sense, such as that the area of Germany was X square miles between 1922 and 1935, and neither Dimension nor Place are E77s.

Next steps are to flesh out examples of Phase, both as text and as graph constructs.

**Comment by annabelleee** [comment]
EXAMPLE USE CASE: The Beverly Hills Main Post Office Building was a US post office from 1934 until 1999, when it was decommissioned as a post office and sold to the City of Beverly Hills. The City then leased the property to the Wallis Annenberg Center for Performing Arts. From 2003 to 2013, the building was under construction as part of a larger development to create a performing arts center. In 2013, the building formally became the entrance lobby to a much larger performing arts center. So, in this case, the use has changed... but also the dimension in a sense? The historic post office building/now lobby has been renamed the Paula Kent Meehan Historic Building, so the historic part that we care about (for now) does have a distinct identity, despite being part of the larger center. If this had happened 100 years ago, then we would also care (for the purposes of a historic place inventory) about the change in use and footprint/volume of the entire center. Thoughts?

**Comment by azaroth42** [comment]
Use Case:  Object Ownership Phase.  The ownership of an object by an actor, that is started by the purchase and ended by its sale. In the intervening time other activities can be part of that phase, such as an art dealer taking inventory or revaluing their stock.
Other aspects could include the validity of identifiers as part of this ownership -- if the owner assigns an accession number, then it is the accession number for that ownership rather than across all time (the next owner might give it a different accession number).

**Comment by azaroth42** [comment]
Use Case: Professional Phase.  For some part of an artist's career they are considered to be a painter, but then they change medium and are later considered to be more of a photographer. The phase would be similar to the building-as-castle vs building-as-hospital case, but instead for a person.

**Comment by Habennin** [comment]
I continue to be troubled by how this would be used. In the case of the building, I feel like we begin to confuse levels of interpretation. So that something was used for x or used for y is something that is not immediately given to the senses but deduced and understood through a chain of evidence (even if that is just a document of an eye witness). If you were to stand before the building, you could very well guess at what it was used for based on typical features but that doesn't make it one of those things. Something that looks like a church morphologically can be functionally a bar. It doesn't seem to be a property of the building itself but more of its social function that it had x or y use. So I wonder if that particular example wouldn't be modelled better as 'use period' or so under E7 and indicate the thing that was used (the building or its part) and what for? It doesn't continuously have to have been used in order for this to be instantiated. 

Professional phase, I cover a lot these days, but I use frbroo F51 Pursuit which is for documenting long term pursuits of some goal by some actor. It seems to cover that ground well. 

Of course I can see and understand the arguments for the need to give temporally bound properties and hence defined phase/state as a reification. What is sketched above is some sort of reification system essentially. Probably neater (though maybe technically more complicated in a non-RDF system) would be a named graph solution. The statement T[A-P-B]1999-2000 where [A-P-B] is the usual CRM triple but as referenced object. State or Phase as modelled above, however, more or less offers this in a way that, perhaps, is easier to implement in Arches?

If so, then my last objection - if no one else has any objections or reservations - is are we sure we are aware of the criterion/a that we would use to pick out the 'state' of a thing. 

So "State of Beverly Hills Main Post Office Building being a Post Office Building" would be the label of the instance? Or, "State of Beverly Hills Main Post Office Building being a Post Office Building from 1934-1999"? What I mean to say is, is the state particular to a time period or is it something that can be re-animated? Real world example would be something like Agia Sophia. If someone turns it back into a mosque (it has been a mosque), is there one state of Agia Sophia being a mosque or two? The same thing would apply for 'box open during photography', 'David Bowie's feminine phase', 'George having a fever'. Is it the case that as soon as fever occurs there are new time spans for my state of having the property 39c temperature? Or do we create a new instance each time I'm sick? My unique fevers?

I think we should be clear on that question of identity. Especially with regards to something like the box being open, it starts to sound like a weird thing to document if each instance of the box being opened is its own distinct phase (in what way does that detail matter to whom?). 

All the ontologic doubt and angst is just that I worry a bunch of non-sense will be put under such a broad term that would be hard to parse after. Ownership case might be an example of that. For building temporal progressions it may be useful to have a state of ownership node sitting around in which one can bundle various things that went on in history, but later on we might properly actually model right holding and various fine properties of such legal activities which might end up conflicting with the documentation work that went on on the premise of the ownership state.

**Comment by azaroth42** [comment]
Use Case: The phase in which a resource has an identifier.  When objects are moved between collections, either between or within organizations, they are frequently given accession numbers.  The accession number is then only valid for a given phase.  That might be related to a collection, or the object might keep the identifier when moved to a different collection in the same organization depending on how the institution manages its data. This would let us connect accession numbers with the period in which they are used, and to the ownership phase of the object (via partitioning).

**Comment by azaroth42** [comment]
Use Case: Objects are given exhibition specific names. The name can be tied to a phase in which it was valid, linked to the exhibition.

**Comment by azaroth42** [comment]
> The statement T[A-P-B]1999-2000 where [A-P-B] is the usual CRM triple but as referenced object. State or Phase as modeled above, however, more or less offers this in a way that, perhaps, is easier to implement in Arches?

Named graphs are also complicated in RDF native systems! Especially as each triple can only be part of one named graph, so it needs to be chosen wisely and hopefully not pre-ordained via external constraints (such as this).

> is the state particular to a time period or is it something that can be re-animated?

I think it's particular to a time period. It has notionally exactly one timespan (even if the dates might not be known). If a castle is used as a restaurant then a hospital then goes back to being a restaurant, then there are two restaurant phases. So, unique fevers.

For the box being open, it matters only for further documentation -- the height of the box while open is X, versus the height of the box while closed is not X. 

I don't follow the objection about ownership. There is a time period in which it is true that `X current_owner Y`. If there's another way to model that in the future, then there could indeed be two ways to say the same thing and we would need to make a choice as to which to use.  But for now, there isn't.

**Comment by azaroth42** [comment]
Use Case:  Existence Phase.  A phase initiated by the Production and terminated by the Destruction of an object.  All other phases for the object could be part of this phase. It doesn't have a relationship or related entity. It's more of a convenience for grouping phases than a real entity of its own.  One could imagine the same for an "all history" period that collects Jurassic, Triassic, Neolithic etc periods.

**Comment by Habennin** [comment]
for identifier case and exhibition case FRBRoo F52

**Comment by Habennin** [comment]
The existence phase is the kind of thing I would worry about, then you don't do events anymore. People can just go back to documenting their pot as a pot.

**Comment by azaroth42** [comment]
How does F52 help with the period over which the name was used, without a second timespan?  The *assignment* of the name Lewis Carroll was done circa 1856 (at the time of his first work), and was carried out by himself, but the name is still *in use* today, long after his death.  He is not still, after his death, carrying out the attribute assignment.

**Comment by Habennin** [comment]
also philosophically accidental being and essential being being represented by the same class also makes me pause.

**Comment by azaroth42** [comment]
Implementation questions for Phase (or Attribute Assignment)

1. How to reference an ontology term from within a Resource Model / Branch.  For example, Attribute Assignment uses P2_has_type to refer to either a concept (an instance of E55 Type) or to an ontology term (eg P4_has_time-span). Current work around is to use domain-value data type to generate a hand curated list of available properties, but it seems like the system could relatively easily compute this based on available knowledge.

2.  For nodes within branches, the directionality of has_phase / phase_of is clear -- it has to go from the branch node to the Phase (has_phase).  For resource model to resource model it is not clear ... it could go from the phase to the (e.g.) man made object, or from the object to the phase.  For consistency it should probably be from the MMO to the Phase. Having a way to then find the inverse from the Phase (what is this a phase of) would be very important for the usability of the Phase as a resource.

**Comment by azaroth42** [comment]
Refactoring idea:

In discussing F52 Name Use Activity, we thought it would be worth exploring if we can reverse engineer the pattern into Attribute Assignment for the case when the carrying out of the activity is ongoing over time by a not-well-identified group of actors.

For example, Charles Dodgson is still called Lewis Carroll long after his death. So that he did the naming in 1856 is a separate attribute assignment from the rest of the world ALSO using that same appellation today.  It did trigger it (O13) however.

Thus:

```
_:pseudonym-adoption a AttributeAssignment ;
  carried_out_by <Charles> ;
  assigned <Name> ;
  assigned_to <Charles> ;
  has_timespan [ 
    a TimeSpan ;
    begin_of_the_begin "1856-01-01" ;
    end_of_the_end "1857-01-01" ] ;
  triggered [
    a AttributeAssignment ;
    carried_out_by <global-group> ;
    assigned <Name> ;
    assigned_to <Charles> ;
    has_timespan [
      a TimeSpan ;
      begin_of_the_begin "1856-01-01" ]
  ]
```

**Comment by azaroth42** [comment]
Use cases I worked through for the Attribute Assignment analysis:

Cases that seem to work well:
* Ownership, Custody - useful as the ownership is not an intrinsic / global, but could be disputed in different jurisdictions. c.f. copyright
* Identification - e.g. in a collection, in an exhibition
* Naming - as identification
* Usage - useful as different uses could be by different groups, potentially simultaneously
* Valuation - very useful to have the valuation limited by group
* Interpretation - e.g. style / genre / subject. ditto
* Profession, Gender, Nationality - All seem reasonable as Attribute Assignment ... but semantic spidey sense starts to twitch.

Cases that are weird:
* Physical characteristics such as size, shape, color.  e.g. Nightwatch was 16' wide, then 14' wide. Is it really an ongoing attribute assignment that it's 14' wide? Who is carrying that out? I can see it as a phase, but less so as an A.A. It starts to be more of a belief, ala CRMInf's I2 (if significantly more useful)
* Existence. Clearly this isn't an attribute assignment. Would be good to see if it can fit into Presence / Spacetime Volume.

**Comment by azaroth42** [comment]
All that said, I question whether F52 is semantically a sub-class of E13. 

* First argument is that the scope note for E13 says:

> In particular, the class describes the actions of people making propositions and statements during
certain museum procedures ...

And F52 (and similarly our phase refactor) is not carried out during a museum procedure. We could propose to the SIG to change "In particular, " to something that makes this an example usage of E13.  The literal interpretation makes E13 very specific.

* Second argument is about the nature of E13's timespan. The F52 interpretation is that it's the span over which the actor holds the assignment to be true. The interpretation that I'm more familiar with is that it's the span at which the assignment was first made, and then might continue to be true after the end_of_the_end.  Someone measuring (e.g. E16) a Painting creates a Dimension, and then goes on to do something else. The Painting continues to have that Dimension after the measurement activity is finished.  The same property shouldn't have different semantics on subclasses.

**Comment by Habennin** [comment]
Just a quick response, because it’s interesting but I don’t have a bunch of time. 

On the one hand, I kind of get the same feeling of semantic inconsistency about the time element, but then thinking about it again, I think maybe there is no problem. In some cases, attribute assignment (including sub classes like measuring) occurs instantaneously and that’s kind of it. I called it that, I moved on. I don’t keep asserting it because it’s not necessary. So the real time span of the real phenomenon is that moment. They called it that on that day and now it is true that someone calls it that. This F52 is more of the performative attribute assignment that we have been mentioning. It is not simply that one day, one called oneself x and that was that, but rather that everyday you get up and be that. So David Paulson everyday gets up and performs being Bono, calling himself that and getting others to do so. So the time span is wider because the activity is wider. Then thinking about dimension. The reason we have dimensions separate to things is that a thing doesn’t have an ultimate dimension, but can be measured at some time and then known to have had a dimension when measured with a certain technique at a certain time. But one can also imagine a measuring activity taking a very long time. I assume some astronomical measurements require much time to execute, so again I think an E16 need not be instantaneous. 

Further thoughts. I agree that the ‘museum procedure’ text is limiting and seems vestigial. It is kept in ostensibly to make the scope notes friendly to museum types…

I have also had the same hesitation though about F52 being under E13. I could equally imagine moving it further up to be a sibling of E13 and have some sort of general ‘use activity’ class that captures acts of ‘using for’.
